lib/style.css

```diff
@@ -460,3 +460,26 @@ button.goog-buttonset-default:hover {
 .material-detail-view .top-tracks {
   background-color: <<BACK_PRIMARY>>;
 }
+
+/*
+* Light theme specific things
+*/
+
+.gmusic-theme-light #material-app-bar {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+.gmusic-theme-light .nav-item-container.selected {
+  color: <<FORE_SECONDARY>>;
+}
+
+body.gmusic-theme-light,
+.gmusic-theme-light .material-detail-view .material-container-details .info .description,
+.gmusic-theme-light .song-table th {
+  color: #212121;
+}
+
+.gmusic-theme-light .song-row .song-indicator {
+  background-image: url('ani_equalizer_black.gif');
+  background-size: 40px 40px;
+}
```

Comment:
It's not clear to me what is dark theme or shared in the CSS. Maybe we should break this up into 3 files:

- common.css
- dark.css
- highlight.css

We should be able to concatenate them in JS when necesssary